src/ast.rs

```diff
@@ -50,6 +92,6 @@ pub enum PrimitiveValue {
     Enumerated(u32),
     // Date, // not yet implemented as they dont have close enough equvalents in rust
     // Time,
-    // ObjectId
+    ObjectId(ObjectType::Type, u32),
```

Comment:
Yeah - everything else is TODO. I put it as a constant as for the moment it just needs that number 8 in there and I'm not yet sure what else will be using the ObjectType.

I picked a complicated way to do something simple and temporary because I didn't know where I was going with it. I think I'll just make it a u8 as it will be a while before there's any other object types